cfgov/v1/models/base.py

```diff
@@ -225,6 +226,12 @@ def get_next_appropriate_siblings(self, hostname, inclusive=False):
     def get_prev_appropriate_siblings(self, hostname, inclusive=False):
         return self.get_appropriate_siblings(hostname=hostname, inclusive=inclusive).filter(path__lte=self.path).order_by('-path')
 
+    def get_context(self, request, *args, **kwargs):
```

Comment:
This is a great approach. I really like the use of Wagtail hooks to customize the context.